bundles/AdminBundle/Resources/public/js/pimcore/settings/web2print.js

```diff
@@ -217,7 +217,7 @@ pimcore.settings.web2print = Class.create({
                         value: t('web2print_headlesschrome_puppeteer_documentation'),
                         autoEl:{
                             tag: 'a',
-                            href: "https://pptr.dev/#?product=Puppeteer&version=v5.2.1&show=api-pagepdfoptions",
+                            href: "https://pptr.dev/api/puppeteer.pdfoptions",
```

Comment:
```suggestion
                            href: "https://github.com/spiritix/php-chrome-html2pdf#options",
```
Leaving here as option, feel free to commit

Reply:
I would go for spiritix options since it adds more options like `pageWaitFor`, `cookies` and so on..